experimenter/experimenter/targeting/constants.py

```diff
@@ -1890,7 +1890,7 @@ def __post_init__(self):
     name="Default Browser FXiOS Users",
     slug="ios_default_browser_user",
     description="Users that already have FXiOS set as the default browser",
-    targeting="is_default_browser == 'true'",
+    targeting="is_default_browser == true && is_first_run",
```

Comment:
Since your targeting requires is_first_run, I would change the name to include `(first run)` or similar

Reply:
Oh just highlighting that `== true` is superfluous here, but shouldn't harm anything.